crowbar_framework/public/docs/cisco_ucs.md

```diff
@@ -75,10 +75,19 @@ All troubleshooting should be done within the Cisco UCS Manager interface.
 
 The UCS Platform Emulator can be [downloaded from Cisco's
 website](http://developer.cisco.com/web/unifiedcomputing/ucsemulatordownload)
-after registering for a free account.  This barclamp has been tested
-with version 2.1(2aPE1) of the emulator.  However at the time of writing, 
-the emulator does not honour power commands, so do not expect this 
-functionality to work.
+after registering for a free account.
+
+Steps to start the UCS Platform Emulator using KVM with virtual cloud on cloud-admin network
```

Comment:
Also, maybe make this line a real sentence: "Here are the steps to... on cloud-admin network:"